### PR TITLE
Add blog post about rollout steps

### LINKED ISSUE #329: Post: write a small post about new dashboard
We want to write a small blog post communicating that we are making the new dashboard the default one. The goal of this blog post is to be simple, short and concrete (can be seen as a changelog entry). We will write another blog post in the future explaining the new features more in details.

* Related: https://github.com/readthedocs/readthedocs-ops/issues/1515#issuecomment-2304642514

### PR DESCRIPTION
**Pull request by agjohnson**
- Fixes #329

----
📚 Documentation preview 📚: https://readthedocs-about--335.org.readthedocs.build/

### CODE REVIEW

File: content/posts/rollout-of-our-new-dashboard.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+:category: Changelog
+:tags: dashboard
+:authors: Anthony Johnson
+:status: published
+:image: /images/new-dashboard.png
+:image_credit: Reviewing build output for a project in our new dashboard
+
+For the last year, you may have seen notifications and hints asking you to try
+out our new dashboard. This new dashboard has been a long term project to
+completely replace our still original user interface with one that is more
```
Comment (ericholscher) [inline content/posts/rollout-of-our-new-dashboard.rst:14]:
```suggestion
completely replace our original user interface with one that is more
```

File: content/posts/rollout-of-our-new-dashboard.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+The new dashboard will start to become the default dashboard
+    After this change, users logging in from our website will be logging into
+    our new dashboard. *Users will still be able to switch back to the legacy
+    dashboard if they need to*.
+
+    - On `Read the Docs Community`_ this change will go into effect on **Nov 12th**
+    - On `Read the Docs Business`_ this change will go into effect on **Dec 10th**
+
+The legacy dashboard will be available until Mar 1st, 2025
```
Comment (ericholscher) [inline content/posts/rollout-of-our-new-dashboard.rst:30]: Do we want to define a date here before we've migrated over? Feels a little ambitious, but probably good to have a deadline, and just do the work to make sure we hit it.

File: content/posts/rollout-of-our-new-dashboard.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+If you are not yet using our new dashboard, here is what you need to know:
+
+The new dashboard will start to become the default dashboard
+    After this change, users logging in from our website will be logging into
+    our new dashboard. *Users will still be able to switch back to the legacy
+    dashboard if they need to*.
+
+    - On `Read the Docs Community`_ this change will go into effect on **Nov 12th**
+    - On `Read the Docs Business`_ this change will go into effect on **Dec 10th**
```
Comment (ericholscher) [inline content/posts/rollout-of-our-new-dashboard.rst:28]: We should mention the URL of the new dashboard here, probably.

File: content/posts/rollout-of-our-new-dashboard.rst [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+added, interfaces that have been updated, and some of the technical changes
+introduced.
+
+Documentation page changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Project documentation will also receive an update with these dashboard changes.
+There are some points in hosted documentation where our own dashboard is
+surfaced, such as documentation error pages, authentication forms, and
+documentation sharing forms.
```
Comment (ericholscher) [inline content/posts/rollout-of-our-new-dashboard.rst:62]:
```suggestion
**The rendered documentation for your project will not change.**
This change primarily applies to the dashboard used for managing your documentation projects,
and not the rendered pages.

There are some points in hosted documentation where our own dashboard is
surfaced, such as documentation error pages, authentication forms, and
documentation sharing forms.
These pages will have the new look and feel.
```

I think we want to be really clear this doesn't impact their actual rendered docs, just system pages.

**Review by ericholscher — APPROVED** [review]
This looks great. Excited to get it out!

File: content/posts/rollout-of-our-new-dashboard.rst (hunk 3 above)
Comment (ericholscher) [reply content/posts/rollout-of-our-new-dashboard.rst:28]:
```suggestion
    - On `Read the Docs Community`_ this change will go into effect on **Nov 12th**
    - On `Read the Docs Business`_ this change will go into effect on **Dec 10th**
    
 
    To log in to the new dashboard:

    - Users of `Read the Docs Community`_ can log in at https://app.readthedocs.org/account/login
    - Users of `Read the Docs Business`_ can log in at https://app.readthedocs.com/account/login
```

File: content/posts/rollout-of-our-new-dashboard.rst (hunk 2 above)
Comment (agjohnson) [reply content/posts/rollout-of-our-new-dashboard.rst:30]: Yeah I felt a date was fine here. The only thing we're waiting on in this period is for users to switch their daily usage to the new dashboard -- we don't need users to make content changes and we won't have any migration tasks left after Dec 10th.

We can also always extend this timeframe if users are stuck using the old dashboard for some reason.

Comment (agjohnson) [reply content/posts/rollout-of-our-new-dashboard.rst:30]: I'll call the date Mar 11th, so 3 months even from Dec 10th.

```suggestion
The legacy dashboard will be available until Mar 11th, 2025
```

File: content/posts/rollout-of-our-new-dashboard.rst (hunk 4 above)
Comment (agjohnson) [reply content/posts/rollout-of-our-new-dashboard.rst:62]: Yeah this is worth spelling out more. I think I'd rather this is more explicit overall now though. It all feels a little vague without this, especially given we're warning the user.

File: content/posts/rollout-of-our-new-dashboard.rst [hunk 5]
```diff
@@ -0,0 +1,90 @@
+:title: Rollout of our new dashboard
+:date: 2024-11-11
```
Comment (humitos) [inline content/posts/rollout-of-our-new-dashboard.rst:2]:
```suggestion
:date: 2024-11-12
```

**Pull request merged**